app/Resources/views/admin/blog/show.html.twig

```diff
@@ -46,5 +46,7 @@
         }, with_context = false) }}
     </div>
 
+    {{ parent() }}
```

Comment:
Probably, the original idea was to show that we can override content of parent block, but I can be wrong here...